### PR TITLE
Retry of emdecr/reskin - Rebrand Alignment

### PR DESCRIPTION
**Pull request by emdecr**
#### Summary
This is a retry PR for #962.
When we merged and pushed to prod, the whole site went wonky.
Even weirder, the preview sites that were rendering before broke after we had already made reviews.

We thought propagation was at play, but that doesn't seem to be it.
Caching, as mentioned, could be at work.

I also capped this before we rolled back, it's looking for the bootstrap stylesheet.
Screen Shot 2022-01-21 at 4 46 50 PM

Thanks,
E

### CODE REVIEW

**Comment by emdecr** [comment]
Hi @jwilander !

Was just chatting with Carrie and I will note:

I did add some temp fixes for the path issues way back when:
https://github.com/mattermost/mattermost-developer-documentation/pull/994/commits/a296701ead2827e5b701159fd6031696d4416ec7#diff-57f64a11732de2fa09163897e01ef8c0026f5f4760458d9041612e2a59b33fa7R55

But removed them shortly before we were going to push live, and that didn't break anything on the preview site:
https://github.com/mattermost/mattermost-developer-documentation/pull/994/commits/a136b5d4c73cd780398ff18d1ce84215ade022b0

BUT I don't think that explains the issues when pushing to prod because what I added should have just thrown errors vs complete breakage....but I could be wrong.

I wonder if there is a cache we need to flush somewhere that we just don't know about. That could explain why smaller **content** changes were fine, but changes to scripts and stylesheets totally broke everything...?

**Comment by jwilander** [comment]
Hi @emdecr, could be a cache issue but I also the preview environment is 404ing on a bunch of assets and code it's trying to pull down.

@cpanato could this potentially be related to the switch to github actions?

**Comment by jwilander** [comment]
I think @spirosoik is taking a look too

**Comment by cpanato** [comment]
in the preview environment or in the prod env?
the preview env is running for quite some time from now.

i did not understand what is the issue

**Comment by cpanato** [comment]
looks like all good now thanks @spirosoik

**Review by jwilander — APPROVED** [review]

File: .gitlab-ci.yml
Comment (cpanato) [inline .gitlab-ci.yml:18]: I think need to set the CGO, check the docs @spirosoik

Comment (cwarnermm) [reply .gitlab-ci.yml:18]: @spirosoik - Following up on this feedback from @cpanato. Is action required for this step prior to merging this into production?

**Comment by cwarnermm** [comment]
@emdecr I've been testing the latest generated preview and stumbled upon some pages that aren't applying the new branding, including:
- [Feature Flags and Mattermost Cloud](https://developers.mattermost.com/contribute/server/feature-flags/)
- [Using Labels to Track Issues and PRs at Mattermost](https://developers.mattermost.com/contribute/getting-started/labels/)
- [Mattermost Core Committers](https://developers.mattermost.com/contribute/getting-started/core-committers/_
- [Managing Inactive Contributions at Mattermost](https://developers.mattermost.com/contribute/getting-started/inactive-contributions/)

This isn't an exhaustive list. Would you prefer I assemble a comprehensive list of issues before merging this PR, or would you prefer to merge this PR now, and then identify the various issues that need further attention?

File: .gitlab-ci.yml
Comment (cwarnermm) [reply .gitlab-ci.yml]: Confirmed with @spirosoik that this has been addressed.

**Comment by cwarnermm** [comment]
Confirmed with @emdecr that the links noted as potentially problematic will have the same new branding as other pages (and that the links in question are likely absolute links rather than relative to the prod environment).

**Review by cwarnermm — APPROVED** [review]
Ready to merge!

**Pull request merged**
